docs/disabled_features.md

```diff
@@ -17,3 +17,9 @@ They may be enabled by default in future versions.
 
 This feature lets you specify the evaluation time for instant vector selectors,
 range vector selectors, and subqueries. More details can be found [here](querying/basics.md#@-modifier).
+
+## Remote Write Receiver
+
+`--enable-feature=remote-write-receiver`
+
+The remote write receiver allows Prometheus to accept remote write requests from other Prometheus servers. More details can be found [here](storage.md#overview).
```

Comment:
It sounds like only Prometheus can write to the Prometheus remote write while I don't think it's a limitation?

```suggestion
The remote write receiver allows Prometheus to accept remote write requests. More details can be found [here](storage.md#overview).
```

Reply:
I would keep as is because e.g. it is not compatible with clients who would send out of order samples (like vmagent).

Reply:
Yeah this is intentionally narrow in scope at the moment.  I'd like to leave as is and consider widening in the future.